templates/repo/view_file.tmpl

```diff
@@ -46,7 +46,10 @@
 			{{end}}
 			{{if not .ReadmeInList}}
 				<div class="ui buttons mr-2">
-					<a class="ui mini basic button" href="{{$.RawFileLink}}">{{.locale.Tr "repo.file_raw"}}</a>
+					<button class="ui mini basic button small compact tooltip" id="clipboard-btn" data-content="{{.locale.Tr "copy"}}" data-fetch-text="true" data-clipboard-target="#raw-file-link" aria-label="{{.locale.Tr "copy"}}">
+						{{svg "octicon-copy" 14}}
+					</button>
+					<a class="ui mini basic button" id="raw-file-link" href="{{$.RawFileLink}}">{{.locale.Tr "repo.file_raw"}}</a>
```

Comment:
Interesting, why is this line marked as changed, even when I ignore whitespace changes?

Reply:
id